pythonforandroid/build.py

```diff
@@ -764,17 +764,15 @@ def run_pymodules_install(ctx, modules, project_dir=None,
             project_has_setup_py(project_dir) and not ignore_setup_py:
         info('Will process project install, if it fails then the '
              'project may not be compatible for Android install.')
-
-    venv = sh.Command(ctx.virtualenv)
+    venv = sh.Command('virtualenv')
     with current_directory(join(ctx.build_dir)):
         shprint(venv,
-                '--python=python{}'.format(
-                    ctx.python_recipe.major_minor_version_string.
-                    partition(".")[0]
-                    ),
-                'venv'
+            '-p python{}'.format(
```

Comment:
If I understand correctly this didn't change, but we only changed `venv = sh.Command('virtualenv')` then maybe don't introduce a change here if we don't end up fixing that line. I mean rollback the full arg `--python=` and the indentation so it's clear what the actual fix is

Reply:
Okay, doing that now...

Reply:
For some reason it gives ```path =python3 not found from --python=python3```.